README.md

```diff
@@ -29,6 +29,12 @@ Argument | Description
 --seq_fasta | Path to the `seq.fasta` file
 --out | Path to the folder where the predictions will be saved
 
+### **NOT required arguments**
```

Comment:
What is the reason to move parameters to separate section? I think, it should be a single list of parameters with description, where description provides information whether this particular parameter is optional.